src/gui/src/mainWindow.cpp

```diff
@@ -1124,6 +1131,13 @@ void MainWindow::showFindDialog()
   find_dialog_->exec();
 }
 
+void MainWindow::showGotoDialog()
+{
+  if (getBlock() == nullptr)
+    return;
+  goto_dialog_->exec();
```

Comment:
when it opens, it should get the current center and margins to help guide the user (you would also be able to get the DBU vs. micron at this point)